python_tests

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+realpath() {
+    [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
+}
+
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
+
+docker run --rm -v "$SCRIPT_DIR":/mubench svamann/mubench nosetests -w mubench.pipeline/
```

Comment:
Please use mubench-ci here, move the file into `mubench.pipeline`, and call it `run_tests`. The `SCRIPT_DIR` should be fixable with an additional call to `dirname`.